### ISSUE TITLE
buggy Docs app on aurelia.io on mobile

### ISSUE THREAD

**Issue by opcodewriter** [aurelia/framework#331]
I went to http://aurelia.io/docs.html on iPad with iOS 8.4.1, after little navigation, tapping on an article title in left pane stops working. 
I can see web-browser navigating but the article content does not load and the small magenta colored indicator does not show near the title I tap in the left pane.
The same issue I saw in the Table of Contents on the right.

I'm sorry I can't give exact reproducible steps, but it shouldn't be hard to see the issues.

**Comment by EisenbergEffect** [comment]
Thanks for catching this. We recently deployed something that created a few issues on mobile. We are working on it.

@joelcoxokc I think there was that bug we discussed in the aurelia-platforms build. I'm not sure if that fixes everything, but when you get a chance, can we fix that issue. I can then deploy and we can do some other device testing to see if anything else is going on. Let me know!
